server/routes/v1/post.ts

```diff
@@ -0,0 +1,77 @@
+import express, {Request, Response} from 'express';
+import db from '../../models';
+import PostController from '../../controllers/v1/post';
+import {User} from '../../models/user';
+import {getAuthUser} from '../../middleware/auth';
+
+const postRouter = express.Router();
+const postController = new PostController(db.User);
+
+postRouter.get('', async (req: Request, res: Response) => {
+  const limit = req.body.limit;
+  const offset = req.body.offset;
+  if (!limit || !offset) {
+    return res.status(400).send({code: 400, message: 'Missing limit or offset'});
+  }
+
+  const result = await postController.getPosts(limit, offset);
+  return res.status(result.status).send(result);
```

Comment:
Should we be using .json to send our result to be consistent with user routes? I believe this also sets the content-type header to json which might be better for us.